.github/workflows/build_deploy.yaml

```diff
@@ -0,0 +1,148 @@
+name: Continuous Integration and Deployment of B2AI Peds Protocol
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set Environment Variables
+      run: |
+        echo "PROTOCOL_COMBINED_SLUG=peds-protocol-combined" >> $GITHUB_ENV
+        echo "PROTOCOL_QUESTIONNAIRES_SLUG=peds-protocol-questionnaires" >> $GITHUB_ENV
+    
+    - name: Echo Environment Variables
+      run: |
+        echo "PROTOCOL_COMBINED_SLUG is set to $PROTOCOL_COMBINED_SLUG"
+        echo "PROTOCOL_QUESTIONNAIRES_SLUG is set to $PROTOCOL_QUESTIONNAIRES_SLUG"
+    
+    - name: Check out repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # Fetches the entire history for comprehensive CI checks
```

Comment:
do you need the entire history? what is using it?

Reply:
removed!